pkg/controller/customdomain/customdomain_controller.go

```diff
@@ -220,6 +220,10 @@ func (r *ReconcileCustomDomain) Reconcile(request reconcile.Request) (reconcile.
 	// such that the record is added to the zone and external DNS can point to it
 	ingressDomain := fmt.Sprintf("%s.%s", instance.Name, dnsConfig.Spec.BaseDomain)
 	ingressName := instance.Name
+	ingressScope := instance.Spec.Scope
+	if ingressScope == ""{
```

Comment:
nit: space between parenthesis
```suggestion
	if ingressScope == "" {
```